indexfile.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project Day 4</title>
+        <link rel="stylesheet" href="stylesheet.css">
+    </head><hr>
+
+  <section class="header">
```

Comment:
- [ ] About myself section is incomplete, please make sure your feature branch is up to date with the main branch. You had to merge the previous pull request to the main branch before creating this branch off the main branch. The portion below is missing. 
![review](https://user-images.githubusercontent.com/65852621/133708562-a5b2bd37-19ea-4abd-adcc-faa4b819f764.png).
Please kindly fix that. 
 The design: 
![design2](https://user-images.githubusercontent.com/65852621/133710050-2e149fee-af93-4856-a2a1-6b206638dd66.png)
Your implementation: 
![yours2](https://user-images.githubusercontent.com/65852621/133710112-4ecdbf31-22b2-42a2-a5f9-c390b503553d.png)